backend/src/main/java/com/benine/backend/camera/BasicCamera.java

```diff
@@ -1,12 +1,7 @@
 package com.benine.backend.camera;
 
-
-import com.benine.backend.Preset;
 import com.benine.backend.video.StreamType;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-
 /**
  * Interface for communication with remote camera's.
  * @author Bryan
```

Comment:
We are working as a team so you should replace Bryan with Benine or remove your name.